superset/assets/src/SqlLab/components/TabbedSqlEditors.jsx

```diff
@@ -197,6 +201,12 @@ class TabbedSqlEditors extends React.PureComponent {
               </div>
               {this.state.hideLeftBar ? t('Expand tool bar') : t('Hide tool bar')}
             </MenuItem>
+            <MenuItem eventKey="4" onClick={this.removeAllOtherQueryEditors.bind(this, qe)}>
```

Comment:
i like this feature, but in most cases we don't bind event handler in render function. see explanation and samples here: https://medium.freecodecamp.org/the-best-way-to-bind-event-handlers-in-react-282db2cf1530. Bind in constructor() is pretty often in Superset code base.

Reply:
Thank you for your reminder, I will fix it today.

Reply:
hi @graceguo-supercat . this method`removeAllOtherQueryEditor()` is in a map function. so it may be not put in constructor().